generator/autogenlist.ts

```diff
@@ -268,10 +484,10 @@ const autoGenList: AutoGenConfig[] = [
         basePath: 'EnterpriseKnowledgeGraph/resource-manager',
         namespace: 'Microsoft.EnterpriseKnowledgeGraph',
     },
-    /*{ NOTE(jcotillo): Temporally removing this RP - latest swagger contains an unsupported type by the schema generator tool
```

Comment:
>latest swagger contains an unsupported type [](http://example.com/codeflow?start=54&length=43)

I recall we had issues when generating this schema, would this break our schema generation?